### ISSUE TITLE
Add to the OWASP project list

### ISSUE THREAD

**Issue by jgadsden** [OWASP/www-project-threat-model-cookbook#3]
This does not appear in the list at https://owasp.org/projects/ - I am not sure why, probably needs a request to go to @hblankenship for this project to be included in the main list

**Comment by jgadsden** [comment]
I think that the problem is in your index.md file where:
```
layout: col-sidebar
title: OWASP Threat Model Cookbook
tags: threatmodeling
level: 0
type: documentation
```

The key for level is 
```
1 - no website yet
2 - incubator project
3- lab project
4 - flagship project
```

at the moment it is set to '0' and so will not (if I understand correctly) appear in the list

I have fixed this in PR #5 , and I think that @hblankenship  does not need to do anything now

**Comment by jgadsden** [comment]
All done in " Fix project level #5 ", thankyou @jmarcil . I can now see it in https://owasp.org/projects/ in the Incubator Projects section
